CMakeLists.txt

```diff
@@ -8,11 +8,11 @@ cmake_minimum_required(VERSION 3.12)
 include(${CMAKE_CURRENT_LIST_DIR}/cmake/HunterGate.cmake)
 set(HUNTER_STATUS_DEBUG ON)
 HunterGate(
-        URL  https://github.com/soramitsu/soramitsu-hunter/archive/9ca72322e8d9de70d360dc7f371b223d32999123.zip
-        SHA1 37cc1150526fb9c5dcf197f918e5d9aab611823f
+    URL https://github.com/soramitsu/soramitsu-hunter/archive/9ca72322e8d9de70d360dc7f371b223d32999123.zip
```

Comment:
not versioned hunter is used

Reply:
Right, after the PR in Kagome is reviewed I will merge this commit in hunter in master and replace the hash here with a tag.